nginx/nginx.plugin.zsh

```diff
@@ -1,5 +1,5 @@
 : ${NGINX_DIR:=/etc/nginx}
-: ${NGINX_VHOST_TEMPLATE:=$ZSH/plugins/nginx/templates/symfony2}
+: ${NGINX_VHOST_TEMPLATE:=$ZSH/custom/plugins/nginx/templates/sitioweb}
```

Comment:
You can set this var in your .zshrc btw, see https://github.com/miklos-martin/oh-my-zsh-plugins#nginx